### ISSUE TITLE
Overwrite folder not being used in Instance mode.

### ISSUE THREAD

**Issue by Astor1977** [TanninOne/modorganizer#955]
I've been using Portable mode for a while, but decided to see what Instance mode was about (tool tip was too vague for me to grasp i suppose).  After pointing it to the appropriate folders, it works just fine.  Except the overwrite folder isn't been read.

MO appears to be writing to it just fine, but doesn't register anything being in the folder, so i have to take the files out and drop them into a mod aptly named OVERWRITE which i keep at the bottom of the left pane.  If i don't, whatever is in the actual overwrite folder isn't registered in MO and doesn't appear in the game.

I've been rolling with it for a couple of days now, have rebooted the pc, double and triple checked that the folder directions are correct (same as Portable which worked just fine).

I'd go back to Portable, but i found the nag screen when launching MO to be tiresome, which is why i clicked on Instance just to see what would happen.  The lack of a nag screen i so very nice. :)

Still loving MO and cannot thank you enough for this program.  :dancer:

**Comment by Gernash** [comment]
it's a bug with folder location 
Conversion guide
http://wiki.step-project.com/User:Gernash/MODWiki#Converting_to_My_wiki_is_easy.....

**Comment by Gernash** [comment]
it's about the MK link 

this is MO2 right?

**Comment by Astor1977** [comment]
Yup, MO2b3.

That guide's workaround for this issue is... quite involved.  Don't know if it's actually worth it to jump through all those hoops, or if i should just wait for a fix or go back to Portable.

Why is that guide so set against using Portable?  It works "out of the box" so to speak whereas Instance apparently does not.

**Comment by Gernash** [comment]
??? move 1 folder and copy past 1 line of text?

**Comment by Gernash** [comment]
yep very involved ;) I just made it steppy so people could feel the awsome

**Comment by Astor1977** [comment]
Was only awake five minutes when i read the guide.  Probably not the best time to do that.  Will go back over it when i'm more alert.

**Comment by Gernash** [comment]
ya is't just move the appdata\mod organizer folder or rename it
run the mklink whatever\appdata\modorganiser mynew folder
run mo2 select profile fo4 exit it
copy all your stuff into the new folders you just made

the WIKI is for users of all level as a guide to nurse them to the end goal it has to be written in a complicated fassion or people will make the booboo

**Comment by Astor1977** [comment]
Got it sorted.  Thanks much.

**Comment by Gernash** [comment]
NP
